source/Engine.h

```diff
@@ -188,6 +188,9 @@ class Engine {
 	// Viewport position and velocity.
 	Point center;
 	Point centerVelocity;
+	Point offset;
+	double offsetMultiplier = 0.;
+	constexpr static double offsetChange = 0.0025;
```

Comment:
You so you know I meant this one as a part of the function, but this is good for me as well